wordpress-mobile/WordPress-iOS
# Login Epilogue: update User Info section

Ref #13413 

This updates the User Info section of the Login Epilogue to the new design. Specifically:

- Top and bottom lines removed.
- Content is left justified.
- New York font used for display name in iOS13.

NOTE: The first label is the Display Name (not Full Name, as shown in the designs).

Not yet updated:
- Bottom buttons.
- iPad specific layout.
- Accessibility.

To test:
- Log into the app.
- Verify User Info looks like the Zeplin design, or that on #13413 .


Light | Dark
--------|-------
![iphone_light](https://user-images.githubusercontent.com/1816888/75182636-05f75a80-56fe-11ea-84db-bbf8788ea575.png)        |       ![iphone_dark](https://user-images.githubusercontent.com/1816888/75182640-0abc0e80-56fe-11ea-8b84-2d071763afaa.png)


PR submission checklist:
